setup-deps.sh

```diff
@@ -153,9 +153,16 @@ npm_install yarn
 test -e ~/.gemrc && grep -q "no-document" ~/.gemrc || echo "gem: --no-document" >> ~/.gemrc
 # rbenv versions | grep -q "2\.3\.3" || rbenv install 2.3.3
 # rbenv versions | grep -q "2\.3\.3" || RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl)" rbenv install 2.3.3
-rbenv versions | grep -q "2\.3\.3" ||  rbenv install 2.3.3
+
+# rbenv versions | grep -q "2\.3\.3" ||  rbenv install 2.3.3
+# eval "$(rbenv init -)"
+# rbenv global 2.3.3
+
+# docker-sync requires Ruby version >= 2.4.
+rbenv versions | grep -q "2\.7\.0" ||  rbenv install 2.7.0
```

Comment:
I'm hesitant about this one until I can test it myself on a fresh machine.

Reply:
I ran into this error on two different machines, so it seems to be an issue. but I understand - I commented it out for now so if someone runs into it, they can uncomment and use the different version